src/components/ResourceList/components/FilterControl/components/FilterValueSelector/tests/FilterValueSelector.test.tsx

```diff
@@ -407,7 +407,7 @@ describe('<FilterValueSelector />', () => {
         expect(operatorsSelect.exists()).toBe(false);
       });
 
-      it('calls onFilterKeyChange when the filter key was changed', () => {
+      it('calls onChange when the filter key was changed', () => {
```

Comment:
We should decide on a convention. Do we use `spy` or the name of the function being called?

Reply:
The name of the function being called IMO. `spy` is a detail of how we do the testing rather than the functionality we want to check.

Also if we call this `spy` then we would violate the [no-identical-title](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-identical-title.md) rule as this test and the test below would both be `calls spy when the filter key was changed` instead of:

`calls onChange when the filter key was changed` and `calls onFilterKeyChange when the filter key was changed`

Reply:
The title should always use the name of the method I would say. In the method you then create a spy named `[name of the method]Spy` to assert the call.

Reply:
I’m good with this too. Agreed that the title should use the name of the method. 

> In the method you then create a spy named [name of the method]Spy (e.g. onChangeSpy) to assert the call.

If you are using `spyOn`, yes. Otherwise, it should be `onChangeMock` if using `jest.fn()`. Although, I find myself preferring to drop the mock or spy suffix for simple tests.